### DISCUSSION TITLE
when will support MacOS 14?

### DISCUSSION THREAD

**Discussion by Hsienjui** [daliansky/XiaoMi-Pro#731] [Q&A]
I updated the beta version of Sonoma and using the newest open core version then the system crashed!Anyone can help with…?And when will support MacOS 14?thx!

**Comment by stevezz3** [comment]
Hi, I am waiting [OpenIntelWireless](https://github.com/OpenIntelWireless) to add support for Sonoma. We need updated [AirportItlwm](https://github.com/OpenIntelWireless/itlwm) to enable wireless network capability on new macOS.

**Comment by ssid1998** [comment]
Has anyone tried updating to MacOS 14 Sonoma (stable) ? Anything to be taken care of while updating?

**Reply by stevezz3** [reply]
Back up the current OS and update EFI before update to Sonoma, same as every major OS update.

**Reply by ssid1998** [reply]
Goes into bootloop. :/

**Reply by stevezz3** [reply]
> Goes into bootloop. :/

Make sure you replaced BOOT and OC folders completely. Delete these two folders first and paste new ones downloaded from this repository's release page.
